### ISSUE TITLE
Guitar Wizard: "Git command" and 'File command" both need text flairs to say "Required".

### ISSUE THREAD

**Issue by tommai78101** [soramimi/Guitar#109]
I am referring to the programs used by Guitar:

![image](https://user-images.githubusercontent.com/6444116/122993164-fed57080-d374-11eb-81be-51bb74b6a881.png)

When running Guitar on first launch, Guitar Wizard will show up.

Someone like me who isn't used to Linux in general, would not know that both `git` and `file` are required.

So if someone put `git` as `/usr/bin/git` but leave `file` blank or empty, Guitar will crash the moment the user exits out of the Guitar Wizard by doing one of the followings:

- The user cancels the Guitar Wizard.
- The user clicks on the Finish button.

There is no error being generated. It just hard crashes.

-----------

Crashes happen for all of these:

- Guitar-79a8603-x86_64.AppImage
- Guitar-1001631-x86_64.AppImage
- Guitar-6003546-x86_64.AppImage

Running on Ubuntu 20.04.02 LTS

------------

Steps to reproduce:

1. In Ubuntu, first, navigate to `~/.config/soramimi.jp/Guitar/`.
2. Do `mv Guitar.ini Guitar.ini.backup` to make a backup copy of your `Guitar.ini` file.
3. Close any running instances of all Guitar apps.
4. Relaunch a new Guitar app. Guitar Wizard should immediately appear.
5. On the `git` command, browse and enter `/usr/bin/git`.
6. Leave `file` blank.
7. Press Next all the way to the end. Skip everything else, as it doesn't matter.
8. Finally, at the end of Guitar Wizard, press either `Cancel` or `Finish`. Either way will crash the app.

**Comment by acarv** [comment]
Yes, it crashes anyway if you follow the suggested file... anyway I have no clue for what that file is or is used for...
Just confirming that GIT was the worst thing that happened in Software Engineering in the last 20 years...
The second worst is the sheer inutility (promoted by GIT also) of having undocumented software...
Its just plain stupid to use a code history manager to perform Configuration Management... I think it will take another 10 years for some people to start understanding that...
